### PR TITLE
feat(cardinal): get /tx/game/txType working

### PR DESCRIPTION
**Pull request by pyrofolium**
Closes: #CAR-194

## What is the purpose of the change

adds route /tx/game/txType 

## Brief Changelog

Should be relatively straightforward builds on the previous issue where I already made utility functions in anticipation for this route. 

## Testing and Verifying

repurposed TestHandleWrappedTransactionWithNoSignatureVerification to test both the old and new route.

### CODE REVIEW

File: cardinal/server/utils.go [hunk 1]
```diff
@@ -183,4 +183,5 @@ func (t *Handler) verifySignatureOfMapRequest(request map[string]interface{}, is
 	} else {
 		return sig.Body, sig, err
 	}
+	return sp.Body, sp, nil
```
Comment (technicallyty) [inline cardinal/server/utils.go:186]: this is unreachable code - i think we can unwrap the else case here?

**Review by technicallyty — COMMENTED** [review]
code looks fine, im a little confused on the title though. i dont think we send readTypes to transaction endpoints?

File: cardinal/server/utils.go (hunk 1 above)
Comment (pyrofolium) [reply cardinal/server/utils.go:186]: Srry this isn't fully ready. I'm merging with previous stuff

**Comment by pyrofolium** [comment]
> code looks fine, im a little confused on the title though. i dont think we send readTypes to transaction endpoints?

Yeah it's actually txType, typo.

File: cardinal/server/server.go [hunk 2]
```diff
@@ -200,16 +200,18 @@ func registerTxHandlerSwagger(world *ecs.World, api *untyped.API, handler *Handl
...
+		if err != nil {
+			return nil, err
+		}
+		tx, err := getTxFromParams("txType", params, txNameToTx)
+		if err != nil {
+			return nil, err
+		}
+		if tx.Name() == ecs.AuthorizePersonaAddressTx.Name() {
+			return nil, errors.New(fmt.Sprintf("This route should not process %s, use tx/persona/%s", tx.Name(), ecs.AuthorizePersonaAddressTx.Name()))
+		}
```
Comment (technicallyty) [inline cardinal/server/server.go:213]: we can just use `fmt.Errorf` here, does the same thing as `errors.New` + `Sprintf`

**Review by technicallyty — APPROVED** [review]
LGTM - one nit

**Pull request merged**